### ISSUE TITLE
Name property of Column not working

### ISSUE THREAD

**Issue by monoflo** [numberoverzero/bloop#125]
Using the `name` property of `Columns` is not working for me.

Here's [the example used in the documentation,](https://bloop.readthedocs.io/en/latest/user/models.html#columns) and its corresponding output for me:

```python
class Impression(BaseModel):
    referrer = Column(String, hash_key=True, name="ref")
    version = Column(Integer, range_key=True, name="v")

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 2, in Impression
TypeError: __init__() got an unexpected keyword argument 'name'
```

**Comment by monoflo** [comment]
I am using Python 3.6.6 and Bloop 2.2.0

**Comment by monoflo** [comment]
Found the issue. The property is now called `dynamo_name`. This isn't stated in the documentation.

Relevant: Line 446 below
https://github.com/numberoverzero/bloop/blob/1234054de94955d6d3c9a3953854018c5d18ded9/bloop/models.py#L434-L447

**Comment by monoflo** [comment]
I've proposed a fix to the documentation to clarify this issue. #126 

[The Public API documentation seems to be consistent](https://bloop.readthedocs.io/en/latest/api/public.html#column), though I don't understand the example for the `name` property. I think it should be moved to the `dynamo_name` property and refactored appropriately.

**Comment by numberoverzero** [comment]
Thanks for identifying the fix **and** opening a PR so fast!

In the public api docs, `name` refers to the Column's property which is computed when the class is defined.  I should add an example to the api docs at least, and maybe the user docs.

In the following, the column has the name `foo` while it has `dynamo_name=bar`:

```python
class MyModel(BaseModel):
    foo = Column(Integer, hash_key=True, dynamo_name="bar")


assert MyModel.foo.name == "foo"
assert MyModel.foo.dynamo_name == "bar"
```

This way, you can reflect over either the name of the column in python, or the name of the column as it's saved in dynamo:

```
for column in MyModel.Meta.columns:
    print(f"{column.name} {column.dynamo_name}")
```

**Comment by numberoverzero** [comment]
Fixed in #126, closing.  Thanks again for the help!
